test/stdlib/KVOKeyPaths.swift

```diff
@@ -216,5 +216,5 @@ testObjectForOptionalKeyPath.optionalObject = "foo"
 
 // CHECK-51-LABEL: observe keyPath with optional value
 // CHECK-51-NEXT: oldValue = Optional(nil), newValue = {{Optional\(nil\)|nil}}
```

Comment:
Might want to apply the same fix to this `Optional(nil)` while we're in here.

Reply:
```suggestion
// CHECK-51-NEXT: oldValue = {{Optional\(nil\)|nil}}, newValue = {{Optional\(nil\)|nil}}
```